docs/apache-airflow/core-concepts/executor/index.rst

```diff
@@ -51,28 +51,27 @@ If you want to check which executor is currently set, you can use the ``airflow
 Executor Types
 --------------
 
-There are two types of executor - those that run tasks *locally* (inside the ``scheduler`` process), and those that run their tasks *remotely* (usually via a pool of *workers*). Airflow comes configured with the ``SequentialExecutor`` by default, which is a local executor, and the safest option for execution, but we *strongly recommend* you change this to ``LocalExecutor`` for small, single-machine installations, or one of the remote executors for a multi-machine/cloud installation.
+Executors are primarily responsible for determining where :doc:`task instances </core-concepts/tasks>` are executed, as well as managing the state of task instances. Executors have a common API and are "pluggable", meaning you can swap executors based on your installation needs.
 
 
 **Local Executors**
 
 .. toctree::
     :maxdepth: 1
 
-    debug
     local
     sequential
 
-**Remote Executors**
+The DebugExecutor also exists but has been deprecated in favor of using the ``dag.test()`` command for :doc:`debugging <../debug>`.
 
-.. toctree::
-    :maxdepth: 1
 
-    celery
-    celery_kubernetes
-    dask
-    kubernetes
-    local_kubernetes
+**Remote Executors**
+
+* :doc:`CeleryExecutor apache-airflow-providers-celery:celery_executor`
+* :doc:`CeleryKubernetesExecutor apache-airflow-providers-celery:celery_kubernetes_executor`
+* :doc:`DaskExecutor apache-airflow-providers-daskexecutor:dask_executor`
```

Comment:
```suggestion
* :doc:`CeleryExecutor <apache-airflow-providers-celery:celery_executor>`
* :doc:`CeleryKubernetesExecutor <apache-airflow-providers-celery:celery_kubernetes_executor>`
* :doc:`DaskExecutor <apache-airflow-providers-daskexecutor:dask_executor>`
```

[Maybe like this example](https://github.com/apache/airflow/blob/9a297380558420cc2f73a409e350f333530e279e/docs/apache-airflow/configurations-ref.rst?plain=1#L38)?

Reply:
jeez, thanks. I guess my eyes glazed over after too much rst